### PR TITLE
Adds nanite machinery to testing lab

### PR DESCRIPTION
**Pull request by 1fbff5f83b23d39d38b1dfcb4cac8d9b**
image

:cl:  McDonald072
rscadd: The Testing Lab now has machines for creating nanites. Go do science!
/:cl:

### CODE REVIEW

**Comment by Nickvr628** [comment]
what do nanites do

**Comment by 1fbff5f83b23d39d38b1dfcb4cac8d9b** [comment]
Cooler version of virology from what I heard

File: _maps/map_files/YogStation/YogStation.dmm [hunk 1]
```diff
@@ -40422,9 +40418,26 @@
...
-/obj/item/integrated_electronics/wirer,
+/obj/item/integrated_electronics/analyzer{
+	pixel_x = -8
+	},
+/obj/item/integrated_electronics/debugger{
+	pixel_x = -8
+	},
+/obj/item/integrated_electronics/wirer{
+	pixel_x = -8;
+	step_x = 0
```
Comment (Amelia0010) [inline _maps/map_files/YogStation/YogStation.dmm]: ew

File: _maps/map_files/YogStation/YogStation.dmm [hunk 2]
```diff
@@ -40422,9 +40418,26 @@
...
+	},
+/obj/item/integrated_electronics/wirer{
+	pixel_x = 8
+	},
+/obj/item/integrated_electronics/debugger{
+	pixel_x = 8
+	},
+/obj/item/integrated_electronics/analyzer{
+	pixel_x = 8;
+	pixel_y = 0
```
Comment (Amelia0010) [inline _maps/map_files/YogStation/YogStation.dmm]: ew

File: _maps/map_files/YogStation/YogStation.dmm [hunk 3]
```diff
@@ -40701,28 +40714,13 @@
...
 "cbZ" = (
-/obj/structure/table/reinforced,
-/obj/machinery/computer/security/telescreen{
-	desc = "Used for watching the RD's goons from the safety of his office.";
-	dir = 1;
-	name = "Research Monitor";
-	network = list("rd");
-	pixel_y = -28
+/obj/machinery/nanite_programmer{
+	step_x = 0
```
Comment (Amelia0010) [inline _maps/map_files/YogStation/YogStation.dmm]: ew

File: _maps/map_files/YogStation/YogStation.dmm [hunk 4]
```diff
@@ -48762,6 +48761,18 @@
...
 /area/engine/atmos)
+"duS" = (
+/obj/machinery/requests_console{
+	department = "Science";
+	departmentType = 2;
+	name = "Science Requests Console";
+	pixel_x = 0;
+	pixel_y = -30;
+	receive_ore_updates = 1;
+	step_x = 0
```
Comment (Amelia0010) [inline _maps/map_files/YogStation/YogStation.dmm]: ew

File: _maps/map_files/YogStation/YogStation.dmm (hunk 1 above)
Comment (1fbff5f83b23d39d38b1dfcb4cac8d9b) [reply _maps/map_files/YogStation/YogStation.dmm]: Crap, I thought those were removed

**Comment by CMOisLing** [comment]
circuits + virology? count me in. I feel like this can still be abused by some robust individuals

**Comment by 1fbff5f83b23d39d38b1dfcb4cac8d9b** [comment]
that was fast

**Comment by Amelia0010** [comment]
image

**Pull request merged**